doc/src/manual/variables-and-scoping.md

```diff
@@ -13,29 +13,16 @@ set of source lines; instead, it will always line up with one of these blocks. T
 main types of scopes in Julia, *global scope* and *local scope*. The latter can be nested. The
 constructs introducing scope blocks are:
 
-# [](@id man-scope-table)
-
-  * Scope blocks that may nest only in other global scope blocks:
-
-    - global scope
-
-      + [`module`](@ref), [`baremodule`](@ref)
-
-      + at interactive prompt (REPL)
-
-    - local scope (don't allow nesting)
-
-      + (mutable) [`struct`](@ref), [`macro`](@ref)
-
-  * Scope blocks which may nest anywhere (in global or local scope):
-
-    - local scope
-
-      + [`for`](@ref), [`while`](@ref), [`try-catch-finally`](@ref try), [`let`](@ref)
-
-      + functions (either syntax, anonymous & do-blocks)
-
-      + comprehensions, broadcast-fusing
+### [Scope constructs](@id man-scope-table)
+
+Construct | Scope type | Scope blocks it may be nested in
+------------ | -------------  |---------------------------
+[`module`](@ref), [`baremodule`](@ref)            | global | other global
+interactive prompt (REPL)                         | global | other global
+(mutable) [`struct`](@ref), [`macro`](@ref)       | local  | other global
+[`for`](@ref), [`while`](@ref), [`try-catch-finally`](@ref try), [`let`](@ref) |local | other local
+functions (either syntax, anonymous & do-blocks) | local | other local
+comprehensions, broadcast-fusing                 | local | other local
```

Comment:
The three last rows here should state `other local or global` as per the old docs I assume?

Reply:
Ah, how could I miss that!! (but I knew those 3 constructs can nest in either global or local scopes). Thanks for checking